app/importers/default_scenario_importer.rb

```diff
@@ -0,0 +1,21 @@
+require 'open-uri'
+class DefaultScenarioImporter
+  def self.import(user)
+    return unless ENV['IMPORT_DEFAULT_SCENARIO_FOR_ALL_USERS'] == 'true'
+    seed(user)
+  end
+
+  def self.seed(user)
+    scenario_import = ScenarioImport.new()
+    scenario_import.set_user(user)
+    default_scenario = File.join(Rails.root, "data", "default_scenario.json")
+    scenario_file = ENV['DEFAULT_SCENARIO_FILE'] || default_scenario
```

Comment:
Sure, then it also guards agains an empty string, instead of just not being set. Thanks